app/packages/core/src/components/Setup.tsx

```diff
@@ -47,7 +47,7 @@ const Code = styled.pre`
 
 const port = (() => {
   if (isElectron()) {
-    return parseInt(process.env.FIFTYONE_SERVER_PORT) || 5151;
```

Comment:
This is used by the Desktop App and should be kept (with priority)
* https://github.com/voxel51/fiftyone/blob/10cf8844dee0a9cea716c745916d80ed901074ec/fiftyone/core/service.py#L415

Reply:
Don't know why we're using a different env var for electron apps, but I bet there's a reason. Addressed in next commit.